visualize-admin/visualization-tool
# fix: Infinite table chart loading

This PR:
- fixes infinite table chart loading by explicitly passing `observationQueryFilters`,
- fixes undefined values labels.

## How to test
**PR**
1. Go to any dataset on deployment preview.
2. See that table chart loads.

**TEST**
1. Go to any dataset on TEST.
3. See that table fails to load.
